data/fwupd.service.in

```diff
@@ -6,6 +6,7 @@ Before=display-manager.service
 
 [Service]
 Type=dbus
+RuntimeDirectory=motd.d
```

Comment:
No prefix?

Reply:
It's implied. Automatically added by whatever you compiled systemd with.

Reply:
Gotcha, thanks.